committee-steering/governance/sig-governance.md

```diff
@@ -1,60 +1,136 @@
-  *SHOULD* be supported by a majority of SIG Members or Subproject Contributors (as applicable).
+- Members of a role *SHOULD* remove any other members that have not communicated
+ a leave of absence and either cannot be reached for more than 1 month or are
+ not fulfilling their documented responsibilities for more than 1 month.
+  This may be done through a [super-majority] vote of members, or if there are
+  not enough *active* members to get a super-majority of votes cast, then
+  removal may occur through a [super-majority] vote between Chairs, Tech Leads
+  and Subproject Owners.
+
+#### Requirements
+- Members *MUST* be [community members] to be eligible to hold a leadership role
+  within a SIG.
+- SIGs *MAY* prefer various levels of domain knowledge depending on the
+role. This should be documented.  
+- People management interests - there's a lot of us!    
+
+#### Escalations
+- Membership disagreements *MAY* be escalated to the SIG Chairs. SIG Chair
+membership disagreements may be escalated to the Steering Committee.
+
+#### New members and Stepping Down
+- Members *MAY* decide to step down at anytime and propose a replacement.  Use
+lazy consensus amongst other members with fallback on majority vote to accept
+proposal.  The candidate *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
+- Members *MAY* select additional members through a [super-majority] vote
+amongst members. This *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
 
 ### Chair
 
 - Chair
-  - Run operations and processes governing the SIG
-  - Number: 2-3
-  - Membership tracked in [sigs.yaml]
+
+- Number: 2-3
+- Membership tracked in [sigs.yaml]  
+  - If no tech lead role is present, Chair assumes responsibilities from #tech-lead
+section in addition  
+Run operations and processes governing the SIG:
+  - *SHOULD* define how priorities and commitments are managed and delegate to
+  other members as needed  
+  - *SHOULD* drive charter changes (including creation) to get community buy-in
+  but *MAY* delegate content creation to community members
+  - *SHOULD* identify, track, and maintain the SIGs enhancements for current
+  release and serve as point of contact for the release team, but *MAY* delegate
+   to another member to fulfill these responsibilities
+  - *MAY* delegate the creation of a SIG roadmap to other members
+  - *MUST* organize a main group meeting and make sure [sigs.yaml] is up to date
+  including subprojects and their meeting information but *SHOULD* delegate the
+  need for subproject meetings to subproject owners  
+  - *SHOULD* facilitate meetings but *MAY* delegate to other members or future
+  chairs/chairs in training
+  - *MUST* ensure there is a maintained CONTRIBUTING.md document in the
+  appropriate SIG folder. *MAY* ask for help from community members to update /
+  send in PRs.
```

Comment:
This differs from the above slightly in that the SIG only needs their own `CONTRIBUTING.md` if it differs from the practices outlined in the contributor guide.

Maybe
```
*MUST* ensure there is a maintained CONTRIBUTING.md document in the appropriate SIG folder *IF*
practices differ the guidelines outlined in the [contributor guide] and [devel] folder. *MAY* ask for help
from community members to update /send in PRs.
```

For reference here is the snippet from the above:
```
- Ensure contributing instructions (CONTRIBUTING.md) are defined in the SIGs
folder located in the Kubernetes/community repo if the groups contributor steps
and experience are different or more in-depth than the documentation listed in
the general [contributor guide] and [devel] folder.  
```